src/shared/directives/conditional-show/conditional-show.directive.ts

```diff
@@ -38,6 +39,10 @@ export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
   }
 
   ngAfterViewInit() {
+    // Ensure this.caseField is actually a CaseField instance even if instantiated with {}
+    if (!(this.caseField instanceof CaseField)) {
```

Comment:
@tzarsmango this same bit of code is duplicated now with the one we introduced the other week.
Can we move this into a proper utility method in one of our Utils classes?

Reply:
👍